### PR TITLE
[ASDisplayNode] Add allowsGroupOpacity property to ASDisplayNode

### PR DESCRIPTION
**Pull request by maicki**
Since we're disabling it in our app, provide easy access for changing the layer's behavior.

Replaces #2310

### CODE REVIEW

**Comment by maicki** [comment]
@levi @Adlai-Holler @appleguy Can I get a look over it please

File: AsyncDisplayKit/Private/_ASPendingState.mm [hunk 1]
```diff
@@ -207,6 +209,7 @@ - (instancetype)init
   contents = nil;
   isHidden = NO;
   needsDisplayOnBoundsChange = NO;
+  allowsGroupOpacity = YES;
```
Comment (levi) [inline AsyncDisplayKit/Private/_ASPendingState.mm]: You don't want to set a default for this, since we've disabled it in our app.

Comment (maicki) [inline AsyncDisplayKit/Private/_ASPendingState.mm]: We have to read the default value from a `UIView` or from the `Info.plist`of the project. Addressed

File: AsyncDisplayKit/Private/_ASPendingState.mm [hunk 2]
```diff
@@ -195,6 +199,12 @@ - (instancetype)init
     CFRetain(blackColorRef);
     CGColorSpaceRelease(colorSpace);
     defaultTintColor = [UIColor colorWithRed:0.0 green:0.478 blue:1.0 alpha:1.0];
+    
+    // Create temporary view to read default values that are based on linked SDK and Info.plist values
+    UIView *view = [[UIView alloc] init];
```
Comment (Adlai-Holler) [inline AsyncDisplayKit/Private/_ASPendingState.mm]: This may not be called on the main thread =/

**Comment by levi** [comment]
@maicki we shouldn't add an extra +load method for this case. Let's just read the info.plist dictionary directly from the bundle.

**Pull request merged**